instrumentation/spring-webclient-5.0/src/main/java/com/nr/agent/instrumentation/spring_webclient/Util.java

```diff
@@ -62,6 +62,7 @@ public void accept(ClientResponse clientResponse) {
                             .uri(uri)
                             .procedure("exchange")
                             .inboundHeaders(new InboundResponseWrapper(clientResponse))
+                            .status(clientResponse.statusCode().value(), "" + clientResponse.statusCode().value())
```

Comment:
Instead of empty string concatenation, we would prefer `String.valueOf(clientResponse.statusCode().value())`.

Reply:
Actually, I think that value could be null.
I don't think it is very useful to duplicate the code as the message, and I don't think Spring webclient provides the response message.

Reply:
Changed to null.